### PR TITLE
slightly nicer?

### PR DESCRIPTION
**Pull request by izaakf**

### CODE REVIEW

File: steely/plugins/markov.py [hunk 1]
```diff
@@ -14,12 +14,11 @@
 
 
 def main(bot, author_id, message, thread_id, thread_type, **kwargs):
-    if random.random() > CHANCE_OF_SEND:
-        return
+    return if random.random() > CHANCE_OF_SEND
```
Comment (byxor) [inline steely/plugins/markov.py]: Hold on. I knew this was prominent in Ruby, but python can do this too???

Comment (izaakf) [reply steely/plugins/markov.py]: uhh kinda, I'm an idiot so what I did there was a ruby thing but [you *kinda can* do it in python](https://docs.python.org/2/reference/expressions.html#conditional-expressions)

Comment (byxor) [reply steely/plugins/markov.py]: That's a shame, I would have adopted it.

**Comment by sentriz** [comment]
thanks sam

**Comment by izaakf** [comment]
I'm retarded is all fellas let's just forget about this PR and move on

**Comment by sentriz** [comment]
it's merged silly boy

**Pull request merged**